test/test-cases/functional/saic/sai-api/test_sai_api_vnet_acl_in.py

```diff
@@ -0,0 +1,96 @@
+import json
+import time
+from pathlib import Path
+from pprint import pprint
+
+import pytest
+
+
+class TestSaiVnetAclIn:
+
+    def test_vnet_acl_in_create(self, dpu):
+
+        commands = [
+            {
+                "name": "acl_in",
+                "op": "create",
+                "type": "SAI_OBJECT_TYPE_DASH_ACL_GROUP",
+                "attributes": [
+                    "SAI_DASH_ACL_GROUP_ATTR_IP_ADDR_FAMILY",
+                    "SAI_IP_ADDR_FAMILY_IPV4"
+                ]
+            },
+        ]
+        results = [*dpu.process_commands(commands)]
+        print("\n======= SAI commands RETURN values create =======")
```

Comment:
Maybe we will need to replace this with logger and let tester choose what to print and what does not by selecting the debug level?

Reply:
This will be added later as a new PR

Reply:
@aputriax A good suggestion, but could you help by showing an example of recommended logger using in sai-challenger framework?

Reply:
@chrispsommers I found a gap in logging configuration. So let's move this change to the future. I'll open an issue in the SAI-C github.